x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/fulltext/FullTextFunctionEvaluator.java

```diff
@@ -39,24 +47,52 @@
  * {@link LuceneSourceOperator} or the like, but sometimes this isn't possible. So
  * this evaluator is here to save the day.
  */
-public class LuceneQueryExpressionEvaluator implements EvalOperator.ExpressionEvaluator {
+public class FullTextFunctionEvaluator implements EvalOperator.ExpressionEvaluator {
```

Comment:
I had some import problems while I was working at this and had to move it outside of compute - I think I can move it  back now and keep the initial name  🤔